extensions/wikia/AchievementsII/Ach_setup.php

```diff
@@ -164,7 +164,8 @@ function Ach_GetHTMLAfterBody($skin, &$html) {
 	global $wgOut, $wgTitle, $wgUser;
 
 	if($wgUser->isLoggedIn() && !($wgUser->getGlobalPreference( 'hidepersonalachievements' ))) {
-		if ($wgTitle->getNamespace() == NS_SPECIAL && array_shift(SpecialPageFactory::resolveAlias($wgTitle->getDBkey())) == 'MyHome') {
+		$spName = SpecialPageFactory::resolveAlias($wgTitle->getDBkey());
+		if ($wgTitle->getNamespace() == NS_SPECIAL && array_shift($spName) == 'MyHome') {
```

Comment:
These two lines should be spacey per coding conventions.

Also we could consider using `$skin->getTitle()` here instead of globals, since `Skin` is a `ContextSource`.